src/combine.jl

```diff
@@ -21,12 +19,33 @@ function merge{T}(ta1::TimeArray{T}, ta2::TimeArray{T}; col_names=[""])
     # combine existing colnames
     cnames = vcat(ta1.colnames, ta2.colnames)
     # check if kwarg to over-ride simple vcat and then if colnames is valid length
-    size(col_names,1) == 1 ? cnames = cnames :       # kwarg not supplied
-    size(col_names,1) == size(vals,2) ? cnames = col_names : error("col_names supplied is not correct size")
+    length(col_names) == size(vals,2) ? cnames = col_names :
```

Comment:
Given the new `Vector` type contraint for `col_names`, I  thought `length(col_names)` would be clearer than `size(col_names, 1)` while remaining unambiguous - I can certainly live with the latter option though if you'd prefer to keep it.